.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,42 +1,50 @@
 GitHub Issue: #
-<!-- Link to relevant GitHub issue if applicable.
-     All PRs should be associated with an issue -->
 
 ## Proposed Changes
-<!-- Please un-comment one ore more that apply to this PR -->
-
-<!-- - Bug fix -->
-<!-- - Feature -->
-<!-- - Code style update (formatting) -->
-<!-- - Refactoring (no functional changes, no api changes) -->
-<!-- - Build or CI related changes -->
-<!-- - Documentation content changes -->
-<!-- - Other, please describe: -->
-
-
-## What is the current behavior?
-<!-- Please describe the current behavior that you are modifying,
-     or link to a relevant issue. -->
-
-
-## What is the new behavior?
-<!-- Please describe the new behavior after your modifications. -->
-
-
-## Checklist
-
-Please check if your PR fulfills the following requirements:
-
-- [ ] Documentation has been added/updated
-- [ ] Automated Unit / Integration tests for the changes have been added/updated
-- [ ] Contains **NO** breaking changes
-- [ ] Updated the Changelog
-- [ ] Associated with an issue
-
-<!-- If this PR contains a breaking change, please describe the impact
-     and migration path for existing applications below. -->
-
+<!-- Please check one or more that apply to this PR. -->
+
+ - [ ] Bug fix
+ - [ ] Feature
+ - [ ] Code style update (formatting)
+ - [ ] Refactoring (no functional changes, no api changes)
+ - [ ] Build or CI related changes
+ - [ ] Documentation content changes
+ - [ ] Other, please describe:
+
+## Description
+
+<!-- Please describe the changes that this PR introduces. -->
+
+
+## Impact on version
+<!-- Please select one or more based on your commits. -->
```

Comment:
Shouldnt you only choose one? Can a pr be considered major and minor at the same time? Or is it per commit?

Reply:
It's not per commit, it's in the scope of the change that you bring on a PR. If any commit is considered as major, then the PR brings major change even if there are other minor or patch commits.

Reply:
then shouldnt it only be "select one" is there ever a case where "or more" is relevant?

Reply:
Hum, you may have a point here. @jeanplevesque what do you think? Especially since all of our open source package has that same typo.

Reply:
Unless what I mentioned in my previous comment is wrong

Reply:
I'm going to keep it as it is for now to keep conformity and avoid confusion. We could probably rectified it later